packages/mwp-core/src/util/cookieUtils.js

```diff
@@ -30,6 +34,14 @@ export const parseBrowserIdCookie = state => {
 	return browserId;
 };
 
+export const parseSiftSessionCookie = state => {
+	if (!state[SIFT_SESSION_COOKIE]) {
+		return '';
+	}
+	const siftSessionId = querystring.parse(state[SIFT_SESSION_COOKIE]);
```

Comment:
Looks like tests are failing because you're using `querystring.parse` on a plain UUID string (it's not querystring-formatted). If you remove that, then the cleanest solution is

```js
export const parseSiftSessionCookie = state => state[SIFT_SESSION_COOKIE] || ''
```

Reply:
Make sense. Update PR as suggested.